packages/notebook/src/notebookfooter.ts

```diff
@@ -0,0 +1,32 @@
+import { Widget } from '@lumino/widgets';
+import { Message } from '@lumino/messaging';
+import { Notebook } from './widget';
+import { NotebookActions } from './actions';
+
+export class NotebookFooter extends Widget {
```

Comment:
Could you add documentation string on this class and its methods.

Reply:
Sure. Documentation strings have been added.